flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/KubernetesOperatorConfigOptions.java

```diff
@@ -244,6 +245,14 @@ public static String operatorConfigKey(String key) {
                     .withDescription(
                             "Maximum number of throwable to be included in CR status error field.");
 
+    @Documentation.Section(SECTION_DYNAMIC)
+    public static final ConfigOption<Map<String, String>> OPERATOR_EXCEPTION_LABEL_MAPPER =
+            operatorConfig("exception.metadata.mapper")
```

Comment:
The config option should also be renamed to `exception.label.mapper` and then please regenerate the docs.